cachetable.go

```diff
@@ -260,7 +260,7 @@ func (table *CacheTable) Exists(key interface{}) bool {
 }
 
 // NotFoundAdd tests whether an item not found in the cache. Unlike the Exists
```

Comment:
Good find! Maybe we should change this line as well: `// NotFoundAdd checks whether an item is not yet cached. Unlike...`